Cargo.toml

```diff
@@ -134,6 +134,7 @@ time = { version = "0.3", default-features = false }
 volatile = "0.6"
 zerocopy = { version = "0.8", default-features = false }
 uhyve-interface = "0.1.3"
+enum_dispatch = "0.3"
```

Comment:
Could you insert this new dependency alphabetically?

Reply:
Done